gtsfm/loader/hilti_loader.py

```diff
@@ -112,6 +114,20 @@ def __load_constraints(self) -> Dict[Tuple[int, int], Constraint]:
         # cast them to dictionary
         return {(constraint.a, constraint.b): constraint for constraint in constraints}
 
+    def _update_stationary_constraints(self, constraints: Dict[Tuple[int, int], Constraint]) -> Dict[Tuple[int, int], Constraint]:
+        updated_constraints = {}
+        TRANSLATION_THRESHOLD = 0.02
+        ROTATION_THRESHOLD = np.deg2rad(0.5)
+        for (a, b), constraint in constraints.items():
+            if np.linalg.norm(Rot3.Logmap(constraint.aTb.rotation())) < ROTATION_THRESHOLD or np.linalg.norm(constraint.aTb.translation()) < TRANSLATION_THRESHOLD:
+                updated_constraints[(a, b)] = constraint
+                continue
+            
+            updated_constraint = Constraint(constraint.a, constraint.b, Pose3(), np.eye(6) * 1e-8, constraint.counts)
```

Comment:
Can you make it not smaller than the rig constraint? Also maybe move it to a global variable above.